webpack.config.js

```diff
@@ -101,3 +89,4 @@ const serverConfig = merge(browserConfig, {
 });
 
 module.exports = [browserConfig, serverConfig];
+module.exports.shared = shared;
```

Comment:
I think this is a little confusing. [The conventional setup](https://webpack.js.org/guides/production/#setup) is to split the common config out in to a _third_ file, instead of munging the main config's exports!
```
@readme/markdown
 ├ webpack.config.js
 ├ webpack.common.js
 └ webpack.dev.js
```